edi_835_parser/loops/claim.py

```diff
@@ -37,7 +37,7 @@ def __init__(
 		self.services = services if services else []
 		self.references = references if references else []
 		self.dates = dates if dates else []
-		self.amount = amount
+		self.amounts = amounts if amounts else []
```

Comment:
How many different amounts do we expect to have here?

Reply:
I think there could be multiple (not sure how many), since its a convention to just use a list to store all possible segments like what we are doing with services, references, dates above